SimG4Core/Geometry/interface/DDDWorld.h

```diff
@@ -29,6 +27,7 @@ class DDDWorld {
   G4VPhysicalVolume *GetWorldVolumeForWorker() const { return m_world; }
 
 private:
+  void SetAsWorld(G4VPhysicalVolume *pv);
```

Comment:
@ianna @civanch the idea behind removing the ```static``` declaration is that in any case this is called only at initialization level, before the event chain?

Reply:
yes. It's DDDWorld self-initialisation - the method became private.